flink-autoscaler/src/test/java/org/apache/flink/autoscaler/JobVertexScalerTest.java

```diff
@@ -379,6 +379,38 @@ public void testParallelismComputationWithAdjustment(
                         maxParallelism,
                         eventCollector,
                         context));
+
+        assertEquals(
+                32,
```

Comment:
It would be nice to tie this answer to something so it was clear why it was picked. 
e.g. 

```suggestion
                EXPECTED_KEY_GROUPS,
```